app/containers/LocaleToggle/index.js

```diff
@@ -5,25 +5,17 @@
  */
 
 import React from 'react';
-import { createSelector } from '@reduxjs/toolkit';
 import { useSelector, useDispatch } from 'react-redux';
 
 import Toggle from 'components/Toggle';
 import Wrapper from './Wrapper';
 import messages from './messages';
 import { appLocales } from '../../locales';
 import { changeLocale } from '../LanguageProvider/slice';
-import { makeSelectLocale } from '../LanguageProvider/selectors';
-
-const stateSelector = createSelector(
-  makeSelectLocale(),
-  locale => ({
-    locale,
-  }),
-);
+import { selectLocale } from '../LanguageProvider/selectors';
 
 export default function LocaleToggle() {
-  const { locale } = useSelector(stateSelector);
```

Comment:
Not super important, but this `{` `}` difference is what inspired this pull request. The relevant tests will fail if `{ locale }` is used instead of `locale` now since `{ locale }` will be `undefined`.